app/tests/test_get_presigned_download_url.py

```diff
@@ -0,0 +1,115 @@
+from unittest.mock import patch, Mock
+
+from botocore.exceptions import ClientError
+
+from hallebarde.get_presigned_download_url import handle
+import hallebarde.config
+
+
+class TestGetPresignedUrl:
+    BUCKET_NAME = f'hallebarde-storage-{hallebarde.config.ENVIRONMENT}'
+    PRESIGNED_URL = 'https://hallebarde-storage-dev.s3.amazonaws.com/test?AWSAccessKeyId=AKEY&Signature=ASign&x-amz-security-token=Atoken&Expires=1581887681'
+
+    @patch('hallebarde.get_presigned_download_url.boto3')
+    @patch('hallebarde.get_presigned_download_url.exchange_repository')
+    @patch('hallebarde.get_presigned_download_url.file_repository')
+    def test_get_presigned_download_returns_a_response_containing_a_presigned_url(self, mock_file_repository,
+                                                                                  mock_exchange_repository,
+                                                                                  mock_boto3,
+                                                                                  download_url_event, an_exchange):
+        # Given
+        mock_client = Mock()
+        mock_client.generate_presigned_url.return_value = self.PRESIGNED_URL
+        mock_boto3.client.return_value = mock_client
+
+        # When
+        response_with_presigned_url: dict = handle(event=download_url_event, context={})
+
+        # Then
+        assert response_with_presigned_url == {
+            "isBase64Encoded": False,
+            "body": self.PRESIGNED_URL,
+            "headers": None,
+            "statusCode": 200
```

Comment:
Just thinking out loud, but in the python standard library, there is the `http` module that proposes an enum with HTTP code status.

Like ```
>>> from http import HTTPStatus
>>> HTTPStatus.OK
<HTTPStatus.OK: 200>
>>> HTTPStatus.OK == 200
True
``` 

I don't know if it could make the routes scenarios more readable, what do you think ?
https://docs.python.org/3/library/http.html

Reply:
Let's do that